### DISCUSSION TITLE
October Check-in

### DISCUSSION THREAD

**Discussion by LiteBrite82** [community/community#141596] [Other Feature Feedback, Questions, & Ideas]
![This banner image shows a outer-space scene with two Ocotcats posing on either side of a rocket ship. It includes the text "Code Search and Navigation check-in" in the upper left corner](https://github.com/user-attachments/assets/7adfbdbd-332f-4359-b64a-9cac036348d2)

Hello GitHub Community, we hope you had a fantastic summer! Let's take a look at what has been happening in CS&N!

## Shoutout to Community Members

**We owe a big thank you to:**

- Thanks to @fiveNinePlusR for finding and reporting a code view bug that was causing the code view to jump around. [Code view jumping around when scrolling. Blame view doesn't have this issue. #135939](https://github.com/orgs/community/discussions/135939). 

- Thanks to @T-Y-G who reported a similar bug with input from @adamlui [Page randomly jumps when scrolling on phone while logged in #131711](https://github.com/orgs/community/discussions/131711).

- Thanks to @daviehh who reported scrolling issues in code view [blank page on google chrome in code view with line number #139047](https://github.com/orgs/community/discussions/139047).

- Thanks to @DavisVaughan who played an integral role in improving Code Search for R with tree-sitter-r by working with the dev team since April of this year! [Better GitHub Code Search for R with tree-sitter-r #120397](https://github.com/orgs/community/discussions/120397). 

- Thanks to @tamdaz who worked with our team to update the coloring of PHP syntax [PHP syntax is not colored correctly. #133592](https://github.com/orgs/community/discussions/133592). 

## 🪐 Universe 2024

As the world’s fair of software, GitHub Universe 2024 will be an unparalleled gathering of the brightest minds, companies, and innovators in the industry. 
* **Get Tickets**: Visit the [Universe website](https://githubuniverse.com/) to purchase your tickets and plan your agenda!  
* **WATCH - Gear up for GitHub Universe 2024**: Want to know more before signing up to GitHub Universe 2024? [Watch our video](https://www.youtube.com/watch?v=Dmchc0dCOa4) to learn about the unique combination of technical discussions, practical solutions, and fun-filled activities we have in store for attendees. 

Will you be attending GitHub Universe? Let us know if you’ll be joining us in person or virtually!

**Comment by JorenVsyntra** [comment]
i dont understand..
